docs/pages/includes/server-access/custom-installer.mdx

```diff
@@ -1,4 +1,26 @@
 {{ cloud="foo" matcher="bar" matchTypes="baz" }}
+To customize an installer, your user must have a role that allows `list`, `create`, `read` and `update` verbs on the `installer` resource.
+
+Create a file called `installercripts-manager.yaml` with the following content:
+```yaml
+kind: role
+version: v5
+metadata:
+  name: installercripts-manager
+spec:
+  allow:
+    rules:
+      - resources: [installer]
+        verbs: [list, create, read, update]
+```
+
```

Comment:
```suggestion
Create the role:

```
To me, it's always a bit jarring when there are multiple successive code fences without some kind of transition or introduction between them.